sql/moz-fx-data-shared-prod/adjust_derived/adjust_derived_v1/query.py

```diff
@@ -0,0 +1,305 @@
+"""Adjust data - download deliverables, clean and upload to BigQuery."""
+import csv
+import json
+import os
+import tempfile
+from argparse import ArgumentParser
+
+import requests
+from google.cloud import bigquery
+
+API_URI = "api.adjust.com"
+ENDPOINT = "kpis"
+API_VERSION = "v1"
+
+"""This is a list of dictionaries.
+Keys are the app_name, and app_token in the form:
+{"app_name":"<specific app name from Adjust dash>". "app_token":"<unique Adjust app token>}"
+Mozilla has 16 apps on Adjust.
+"""
+
+APP_TOKEN_LIST = os.environ.get("ADJUST_APP_TOKEN_LIST")
+
+"""This is Marlene's personal Adjust API Token. We don't have a special service account API Token."""
```

Comment:
```suggestion
```

This piece of logic is not dependent on who's credentials are in the env variable. I'd consider removing this comment as it could quickly become out of date and misleading.

Reply:
George recommended that I make a note as to whose credentials are being used. If in future we have to make changes and update/take out the comment, it can be done then

Reply:
I've made note of the use of my API token in Airflow. I'm just reiterating that we are using my token here as a way to remind people/future people

Reply:
Sounds good. Will move it